libs/selectors/src/Select.tsx

```diff
@@ -409,7 +409,7 @@ function Select<T>({
         >
           <div className={tokens.Button.container}>
             <div className={tokens.Button.value}>{valueLabel}</div>
-            <input className={tokens.Button.input} ref={inputRef} tabIndex={-1} />
+            <input name={name} className={tokens.Button.input} ref={inputRef} tabIndex={-1} />
```

Comment:
why is this in this PR ? If you found another error, please create a separate issue and PR